selfdrive/car/honda/interface.py

```diff
@@ -387,7 +387,7 @@ def update(self, c):
     ret.wheelSpeeds.rr = self.CS.v_wheel_rr
 
     # gas pedal
-    ret.gas = self.CS.car_gas / 256.0
+    ret.gas = self.CS.pedal_gas / 256.0
```

Comment:
What is this used for?

Reply:
as discussed, let's keep the definition of car_gas as the gas applied either by the driver or by ACC, whenever possible. 
I'll work on a restructure that more clearly separates between pedal_gas and car_gas.